src/main/java/mate/academy/until/ConnectionUtil.java

```diff
@@ -0,0 +1,28 @@
+package mate.academy.until;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+
+public class ConnectionUtil {
+    private static final String JDBC_DRIVER = "com.mysql.cj.jdbc.Driver";
+    private static final String DB_URL = "jdbc:mysql://localhost:3306/academy";
+    private static final Properties DB_PROPERTIES;
+
+    static {
+        DB_PROPERTIES = new Properties();
+        DB_PROPERTIES.put("user", "root");
+        DB_PROPERTIES.put("password", "");
```

Comment:
Consider externalizing database credentials to a configuration file or environment variables for better security practices, especially if this code is used in a production environment.